html/janus.js

```diff
@@ -613,6 +613,10 @@ function Janus(gatewayCallbacks) {
 	this.destroyOnUnload = true;
 	if(typeof gatewayCallbacks.destroyOnUnload !== 'undefined' && gatewayCallbacks.destroyOnUnload !== null)
 		this.destroyOnUnload = (gatewayCallbacks.destroyOnUnload === true);
+	// Whether we should try the other servers when the websocket closed abnormally or the HttpApiCall fails
+	let autoTryOtherServers  = true;
+	if(typeof gatewayCallbacks.autoTryOtherServers  !== 'undefined' && gatewayCallbacks.autoTryOtherServers  !== null)
+		autoTryOtherServers  = (gatewayCallbacks.autoTryOtherServers  === true);
```

Comment:
Small nit: I see some double spaces in the lines above, where just one would suffice.